src/controllers/engine/controllerengine.cpp

```diff
@@ -380,7 +380,7 @@ void ControllerEngine::scriptErrorDialog(const QString& detailedError) {
     ErrorDialogProperties* props = ErrorDialogHandler::instance()->newDialogProperties();
     props->setType(DLG_WARNING);
     props->setTitle(tr("Controller script error"));
-    props->setText(tr("A control you just used is not working properly."));
+    props->setText(tr("The %1 controller is not working properly.").arg(m_pController->getName()));
```

Comment:
Good idea to specify the controller here. However, the error message could mislead users to think their hardware is faulty. How about: "The mapping for the %1 controller is not working properly"

Reply:
Or we could say "script" instead of "mapping" :man_shrugging:

Reply:
Both seem fine to me, whatever you prefer :)